### ISSUE TITLE
ModuleNotFoundError: No module named 'craft_cli'

### ISSUE THREAD

**Issue by AurelienLourot** [canonical/charmcraft#617]
The latest commit on `master` fails with

```
Traceback (most recent call last):
  File "/home/lourot/Documents/git/canonical/charm-nova-compute-nvidia-vgpu/.tox/build/bin/charmcraft", line 5, in <module>
    from charmcraft.main import main
  File "/home/lourot/Documents/git/canonical/charm-nova-compute-nvidia-vgpu/.tox/build/lib/python3.9/site-packages/charmcraft/main.py", line 24, in <module>
    from craft_cli import emit, EmitterMode, CraftError
ModuleNotFoundError: No module named 'craft_cli'
```

I believe a dependency to https://github.com/canonical/craft-cli is missing

**Comment by facundobatista** [comment]
How are you installing/testing `charmcraft`?

The `craft-cli` tool is declared in the `requirements.txt` file...

**Comment by AurelienLourot** [comment]
I am pulling charmcraft from PyPI in my project's `requirements.txt`. To get it to work I end up having to do all this:

```
# NOTES(lourot):
# * Workaround for getting python-apt on Impish+, see
#   https://github.com/canonical/charmcraft/issues/615
# * This is a fork of https://salsa.debian.org/apt-team/python-apt , which
#   isn't accessible from our CI system without proxy.
# * Requires libapt-pkg-dev to build.
git+https://github.com/AurelienLourot/python-apt.git#egg=python-apt

# NOTE(lourot): the latest version published to PyPI (charmcraft 1.3.2) fails
# in a way similar to https://github.com/canonical/charmcraft/issues/587
# Thus pulling from the latest sources until a new version gets published:
git+https://github.com/canonical/charmcraft.git#egg=charmcraft

# NOTE(lourot): workaround for
# https://github.com/canonical/charmcraft/issues/617
git+https://github.com/canonical/craft-cli.git#egg=craft-cli

# NOTE(lourot): workaround for
# https://github.com/canonical/craft-parts/issues/158
pydantic-yaml==0.4.2
```

**Comment by facundobatista** [comment]
I updated the README file specific instructions on how to install from PyPI. I tested those with version 1.5 and they work ok. Please refer to that, reopen this if you find any problem.

Thanks!
